cli/node.ts

```diff
@@ -0,0 +1,24 @@
+import { Buffer, readLines } from "../deps/std/io.ts"
+import { readerFromStreamReader, writeAll } from "../deps/std/streams.ts"
+
+export interface NodeRunFlags {
+}
+
+export default function run({}: NodeRunFlags) {
```

Comment:
What can we share between the Deno and Node.JS `run`s?

Reply:
Seemingly everything beneath the spawning of the process can be reused between the two?